### ISSUE TITLE
Documentation error about Files member of ContentTypes enum

### ISSUE THREAD

**Issue by guzelonur** [Willy-Kimura/SharpClipboard#25]
Hi,

I found out that the documentation note for "Files" member of ContentTypes enum appears to be explained as: "Represents content as a List <**string**> of files" at line 102 in [SharpClipboard.cs](https://github.com/Willy-Kimura/SharpClipboard/blob/6584fccb08de7d039074cba4c96dd3f59630e5b6/SharpClipboard/SharpClipboard.cs) file.

However it's obvious that if you assign the the value "Files" to a generic List of strings, you get a InvalidCastException. However, if you get the content to a classic string array, it's just fine.

So, I think the return type of ContentTypes for Files member is quite misleading and the documentation text should be corrected as "Represents content as **string array**", at line 102 which would be fine :)

Best regards.

**Comment by MontanaCode** [comment]
Yes ... it gives an array of strings representing FilePath\Names.

**Comment by Willy-Kimura** [comment]
Great thanks @guzelonur. Will be sure to update the docs and summary notes.
